### PR TITLE
docs(es): Dec 10 - Dec 13

### PR DESCRIPTION
**Pull request by PedroMidueno**
This PR translates the commits from [Dec 10 - Dec 13](https://github.com/vuejs/docs/commits/main?branch=main&qualified_name=refs%2Fheads%2Fmain&since=2023-12-10&until=2023-12-13)

> [!NOTE]
> The commits [docs: fix minor grammatical mistake](https://github.com/vuejs/docs/commit/4151f8276330454e6c9866efcced8ea2e3675d54) and [Fix terminology and grammar](https://github.com/vuejs/docs/commit/c0f3490a654f4f18a819a73645334b9bf59d3ec2) were not added as they fixed grammatical errors in English that are not present in the Spanish translation.

### CODE REVIEW

File: src/guide/scaling-up/testing.md [hunk 1]
```diff
@@ -125,76 +125,68 @@ Las pruebas de componentes deben centrarse en las interfaces públicas del compo
...
-</div>
+  ```js
+  const { getByText } = render(Stepper, {
+    props: {
+      max: 1
+    }
+  })
 
-<div class="cypress-api">
+  getByText('0') // Afirmación implícita de que "0" está dentro del componente
```
Comment (mrcego) [inline src/guide/scaling-up/testing.md]: ¿Qué diferencia hay entre aserción y afirmación?

File: src/about/coc.md [hunk 2]
```diff
@@ -42,4 +42,6 @@ Los encargados del mantenimiento del proyecto que no sigan o hagan cumplir el C
 
 Este Código de Conducta es una adaptación del [Contributor Covenant][homepage], versión 1.4, disponible en https://www.contributor-covenant.org/version/1/4/code-of-conduct.html
 
+Para obtener respuestas a preguntas comunes sobre este código de conducta, consulte https://www.contributor-covenant.org/faq
```
Comment (mrcego) [inline src/about/coc.md]: Lenguaje formal: Para obtener respuestas a preguntas comunes sobre este código de conducta, **consulta** ...

File: src/guide/scaling-up/testing.md (hunk 1 above)
Comment (icarusgk) [reply src/guide/scaling-up/testing.md]: "Aserción" pienso que sería mejor en este contexto de testing

File: src/about/coc.md (hunk 2 above)
Comment (icarusgk) [reply src/about/coc.md]: Si, en los docs estamos tuteando @PedroMidueno

**Review by mrcego — APPROVED** [review]

**Review by glopzel — APPROVED** [review]

**Review by icarusgk — APPROVED** [review]
LGTM

**Pull request merged**